interface/main/messages/print_postcards.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * /library/MedEx/print_postcards.php
+ *
+ * This file is executed as a background service
+ * either through ajax or cron.
+ *
+ * Copyright (C) 2017 MedEx <magauran@MedExBank.com>
+ *
+ * LICENSE: This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *  Portions of this were developed using Terry Hill's addr_label code.
+ *
+ * @package OpenEMR
```

Comment:
Currently via this code the postcards are printed locally within openEMR on a local printer.  The option to send a file to MedEx for postcards remotely does not exist yet.  Should it be @package openEMR or MedEx?  You decide.  I really don't think it is important but the idea is at this juncture is that this code is for openEMR and does not need MedEx (or any other entity that wants to implement this).